plugins/woocommerce/src/Internal/Admin/ReviewsListTable.php

```diff
@@ -1390,4 +1384,97 @@ class="wc-product-search"
 		<?php
 	}
 
+	/**
+	 * Displays a review count bubble. Based on {@see WP_List_Table::comments_bubble()}, but overridden so we can
+	 * customize the URL and some of the language.
+	 *
+	 * @param int $post_id          The product ID.
+	 * @param int $pending_comments Number of pending reviews.
+	 */
+	protected function comments_bubble( $post_id, $pending_comments ) {
```

Comment:
I intentionally left these variable names still referencing "post" and "comment" to keep them matching the parent class's method signature.